### PR TITLE
4291 validate trading pair

### PR DESCRIPTION
**Pull request by ducquangkstn**

### CODE REVIEW

File: v3/http/trading_pair_create.go [hunk 1]
```diff
@@ -27,6 +36,44 @@ func (s *Server) createCreateTradingPair(c *gin.Context) {
...
+	quote, err := s.storage.GetAsset(createEntry.Quote)
+	if err != nil {
+		return errors.Wrapf(common.ErrBaseAssetInvalid, "quote id: %v", createEntry.Quote)
+	}
+
+	if !quote.IsQuote {
+		return errors.Wrap(common.ErrQuoteAssetInvalid, "quote asset should have is_quote=true")
+	}
+
+	containExchange := false
```
Comment (secmask) [inline v3/http/trading_pair_create.go]: this block can be extract to function

File: v3/http/trading_pair_create.go [hunk 2]
```diff
@@ -27,6 +36,44 @@ func (s *Server) createCreateTradingPair(c *gin.Context) {
...
+	for _, exchange := range base.Exchanges {
+		if exchange.ExchangeID == createEntry.ExchangeID {
+			containExchange = true
+		}
+	}
+	if !containExchange {
+		return errors.Wrap(common.ErrBaseAssetInvalid, "exchange id not found")
+	}
+
+	containExchange = false
```
Comment (secmask) [inline v3/http/trading_pair_create.go]: this finding block can also extract into function

File: v3/http/trading_pair_create.go (hunk 1 above)
Comment (ducquangkstn) [reply v3/http/trading_pair_create.go]: change as request

File: v3/http/trading_pair_create.go (hunk 2 above)
Comment (ducquangkstn) [reply v3/http/trading_pair_create.go]: change as request

**Pull request merged**